include/psp2kern/kernel/threadmgr/debugger.h

```diff
@@ -87,13 +87,16 @@ int ksceKernelGetThreadIdList(SceUID pid, SceUID *ids, int n, int *copy_count);
 int ksceKernelGetThreadCpuRegisters(SceUID thid, SceThreadCpuRegisters *registers);
 
 /**
- * @brief      Call from an abort handler to get info on faulting process
+ * @brief      Obtain the context information for the thread scheduled on this CPU.
  *
- * @param      info  Output info
+ * This function can be called from an exception handler to obtain information about
+ * the thread that was interrupted.
  *
- * @return     Zero on success
+ * @param[out]  pInfo   Context information of the thread
+ *
+ * @return      Zero on success, < 0 on error
  */
-int ksceKernelGetFaultingProcess(SceKernelFaultingProcessInfo *info);
```

Comment:
I think it should keep and mark as `deprecated` to support backward compatibility.
@Princess-of-Sleeping can you review this?

Reply:
Yeah, Or make macro is good.